x=1
try:
    print(x)
except NameError:
    print("Invalid value")